articles/azure-sql/database/connect-query-python.md

```diff
@@ -51,9 +51,9 @@ To further explore Python and the database in Azure SQL Database, see [Azure SQL
    ```python
    import pyodbc
    server = '<server>.database.windows.net'
-   database = '<database>'
+   database = 'tcp:<database>'
```

Comment:
this is the database field, as mentioned in comment the protocol prefix goes on the server field

Reply:
thank you @dzsquared  I have fixed as per suggestion.
Do review.